charge_point_gateway_format.proto

```diff
@@ -64,6 +64,10 @@ message EVChargingStatusData {
   string battery_unit = 4;
```

Comment:
Every field that has "battery" should be removed as agreed.

Reply:
Removed